.changeset/two-tables-do.md

```diff
@@ -4,7 +4,7 @@
 "@salt-ds/theme": minor
 ---
 
-Removed differential characteristic
+Deprecated differential characteristic
 Moved differential tokens to status characteristic
 Added 'static' status variant
 
```

Comment:
might be worth changing the diff for something like: 

use `--salt-status-positive-foreground` instead of `--salt-differential-positive-foreground`.
use `` instead of `` ...